### PR TITLE
Improved readme accessibility and usability

### PR DESCRIPTION
**Pull request by victoriaquasar**
## PR Summary

Implemented a summary in the repository readme to ease getting information and improve the readability. It will also be useful to improve the accessibility of screen readers, since in addition to the table of contents, options to return to the table of contents have been implemented at important points where information is finalized.

## PR Checklist

- [x] PR has a meaningful title
- [x] Summarized changes
- [x] Change is not breaking
- [x] This PR is ready to merge and is not **Work in Progress**

### CODE REVIEW

**Comment by victoriaquasar** [comment]
> @victoriaquasar please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="default"]

File: README.md [hunk 1]
```diff
@@ -5,7 +5,28 @@ PSRule works great and integrates with popular continuous integration (CI) syste
...
-Features of PSRule include:
+### Summary
+
+- [Introduction](#summary)
+- [Project Objectives](#project-objectives)
+- [Support](#support)
+- [Getting the module](#getting-the-module)
+- [Getting extensions](#getting-extensions)
+- [Getting started](#getting-started)
+  - [Scenarios](scenarios)
```
Comment (BernieWhite) [inline README.md:16]:
```suggestion
  - [Scenarios](#scenarios)
```

**Review by BernieWhite — CHANGES_REQUESTED** [review]
@victoriaquasar Thanks for the PR. Looks great. One small typo. After that all good to merger.

File: README.md (hunk 1 above)
Comment (victoriaquasar) [reply README.md]: Fixed!

**Review by BernieWhite — APPROVED** [review]
@victoriaquasar Great work! all good to merge.

**Pull request merged**